### DISCUSSION TITLE
Creation of GPU nodepool with OS ubuntu latest version(2204) with --aks-custom-headers UseGPUDedicatedVHD=true

### DISCUSSION THREAD

**Discussion by msgurikar** [Azure/AKS#3727] [General]
Hi,
Do we have support GPU node pool with latest Ubuntu version(2204) and NVIDIA driver already installed in the image.

We tried this command, but its creating Ubuntu 1804 version 

az aks nodepool add  --resource-group my-rg-app --cluster-name k8s-dev-aks --name gpunp4 --node-count 1 --node-vm-size Standard_NC6s_v3 --node-taints sku=gpu:NoSchedule --labels algo=linux-gpu-nc6s-v3-2204  --enable-cluster-autoscaler --os-type Linux --min-count 0 --max-count 5 --aks-custom-headers UseGPUDedicatedVHD=true

Thank you.

**Comment by samrui** [comment]
I have the same problem, any update ?
